Pulse/FairPlayPulsePlayer/objc/FairPlayPulsePlayer/Library.json

```diff
@@ -1,6 +1,4 @@
 [{
-    "content-title":     "No ads."
```

Comment:
@cacel would be good to add this case "Extend with Standard-Midrolls."

Reply:
Added, also PauseAds